spidermon/contrib/actions/slack/__init__.py

```diff
@@ -105,7 +105,9 @@ def _api_call(self, method, **kwargs):
         return response
```

Comment:
**suggestion** This line added some validation that was required by older versions of `slackclient`. Maybe it may be removed (or updated):

```if isinstance(response, six.string_types):  # slackclient < v1.0```